README.md

```diff
@@ -98,142 +113,140 @@ oc login -u developer -p developer https://localhost:8443
-```
+We will create the bindings to each of the application as we deploy them.
 
 ### Deploying the "Temperature Analyzer" Spark driver
 
-The `spark-driver` directory provides the Spark Streaming driver application and a Docker image for running the related Spark driver inside the cluster. Using `minishift`, we need that the following Docker images (built from the source) will be available in the local `minishift` Docker registry. For this reason, it's needed to run the following command first :
+The `spark-driver` directory provides the Spark Streaming driver application and a Docker image for running the related Spark driver inside the cluster. The spark-driver is deployed by building and running it on the OpenShift cluster.  The spark-driver uses the [fabric8-maven-plugin](https://github.com/fabric8io/fabric8-maven-plugin) to create a docker image, an OpenShift deployment config, and deploy the spark-driver into OpenShift.
+
+To deploy the spark driver:
 
 ```
-eval $(minishift docker-env)
+mvn clean package fabric8:resource fabric8:build fabric8:deploy -Dspark.master.host=myspark.spark.svc -Dspark.app=myapp
 ```
 
-In order to set as `DOCKER_HOST`, the `minishift` environment and all the Docker images will be built using that environment and will be available in `minishift` Docker registry.
+This command will package the application and build a Docker image deployed to OpenShift.
 
-This application can be packaged running the following command from such directory.
+Once the driver has been deployed, we need to create a binding with the permissions we defined above. Click on "Create binding" to open the dialog to create a binding. Set `sendAddresses` to `max` and `recvAddresses` to `temperature`.
 
-```
-mvn clean install package -Pbuild-docker-image
-```
+![Binding1](images/binding1.png)
 
-This command will package the application and build a Docker image ready to be deployed on OpenShift.
-In order to deploy the Spark driver, an OpenShift template is available which can be instantiated with the following command:
+Go to the secret that was created and click "Add to application". This will allow you modify your application deployment to mount the secret so that the example application can use it. Select the option to mount it and enter `/etc/app-credentials` as the mount point.
 
-```
-cd iot/spark-driver
-oc process -f target/fabric8/spark-driver-template.yaml SPARK_MASTER_HOST=$SPARK_NAME SPARK_DRIVER_USERNAME=<user> SPARK_DRIVER_PASSWORD=<password> | oc create -f -
-```
-It's possible to configure the Spark driver changing these parameters:
 
-* _SPARK_MASTER_HOST_ : hostname of the Spark master node
-* _SPARK_MASTER_PORT_ : the port of the Spark master node (default value is 7077)
-* _SPARK_DRIVER_USERNAME_ : username provided by Keycloak for driver authentication with EnMasse
-* _SPARK_DRIVER_PASSWORD_ : password provided by Keycloak for driver authentication with EnMasse
+![Secret2](images/secret2.png)
```

Comment:
There is no secret2.png file in the images folder.

Reply:
fixed